src/components/Lobbies.js

```diff
@@ -20,12 +21,18 @@ const Lobbies = ({ lobbies, loggedUser }) => {
       <a href={`/lobby/room/${lobby.lobby_id}`} key={index} onClick={joinLobby}>
         <div className="lobbies__lobby flex flex-row items-center">
           <div className="lobbies__info flex flex-row items-center">
-            <img src="/img/critLogoph.svg" alt="" />
+            <div className="lobbies__image relative">
+              <img src="/img/critLogoph.svg" alt="" />
+              { personality.personality_type != undefined 
```

Comment:
`!= undefined` is the same as just leaving it blank
```suggestion
              { personality.personality_type
```